cfgov/cfgov/urls.py

```diff
@@ -37,16 +37,16 @@
 
     url(r'^home/(?P<path>.*)$', RedirectView.as_view(url='/%(path)s', permanent=True)),
 
-    url(r'^owning-a-home/static/(?P<path>.*)$', RedirectView.as_view(url='/static/owning-a-home/static/%(path)s')),
-    url(r'^owning-a-home/resources/(?P<path>.*)$', RedirectView.as_view(url='/static/owning-a-home/resources/%(path)s')),
+    url(r'^owning-a-home/static/(?P<path>.*)$', RedirectView.as_view(url='/static/owning-a-home/static/%(path)s', permanent=True)),
```

Comment:
In Django 1.9 the default value of permanent for [RedirectView](https://docs.djangoproject.com/en/1.9/ref/class-based-views/base/#redirectview) changes from false to true. If you do not specify a value for this you get RemovedInDjango19Warnings when testing against this file. This change just makes explicit what is (in the current version of Django 1.8) currently implicit.